docs/modules/ROOT/pages/how-to-guides/testing_applications/retriggering_tests.adoc

```diff
@@ -0,0 +1,13 @@
+= Retriggering Tests
```

Comment:
```suggestion
= Retriggering Integration Tests
```

Reply:
Fixed, I also renamed the file to match the new title.